redis-utils-lib/src/cmr/redis_utils/config.clj

```diff
@@ -35,13 +63,40 @@
 
 (defconfig redis-retry-interval
   "The number of milliseconds to wait between Redis retry command."
-  {:default 100 :type Long})
+  {:default 400 :type Long})
```

Comment:
why 4 times longer?

Reply:
I logged how long it takes these connections.  While most of these transactions are under 100 milliseconds, I found that some times these transactions take 100-300 milliseconds, so just by increasing the timeout, I reduce the number of requests to redis during peak load.  When we re-try redis still has to finish the original task, but then we ask it to do it again, and if that one takes longer than 100 milliseconds we ask it to do it again, so under peak load it really adds un-necessarily more work.

Reply:
can you summarize this into a 1 sentence comment to warn a future developer from randomly making the value shorter then 400?